woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php

```diff
@@ -489,7 +489,7 @@ public function get_currency_rate($local_currency_code) {
 	 */
 	protected function request_data($order)
 	{
-		$home_url = esc_url( home_url() );
+		$home_url = esc_url( rtrim ( home_url(), '/' ) . '/' );
```

Comment:
I think `rtrim` is unnecessary because `esc_url` do a good work. Take a look: https://core.trac.wordpress.org/browser/tags/4.7.3/src/wp-includes/formatting.php#L3767

What do you think?
```
$home_url = esc_url( home_url( '/' ) );
```